libbeat/docs/breaking-7.4.asciidoc

```diff
@@ -0,0 +1,47 @@
+[[breaking-changes-7.4]]
+
+=== Breaking changes in 7.4
+
+This section discusses the main changes that you need to be aware of to
+migrate Beats to version 7.4. {see-relnotes}
+
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+//tag::notable-breaking-changes[]
+
+[float]
+==== Golang version update
+
+The Beats platform has been updated to use Golang 1.12.7.
+
+// REVIEWERS: Is the above statement correct (taken from the relnotes)? The 
```

Comment:
@mikemadden42 Thanks for the review! I do still have an open question, though. Is this statement correct? If so, why does the golang version shown here conflict with the version info in version.asciidoc?

Reply:
@mikemadden42 I'm using the version that's shown in version.asciidoc (1.12.9). Let me know if that's not correct.